core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -120,23 +128,27 @@ export class TranslateTextService {
     );
   }
 
+  // The following property is set to null
+  // when there is no text to translate.
   private _getUpdatedTextToTranslate(
-      text: string | string[],
+      text: string | string[] | null,
       more: boolean,
       status: Status,
-      translation: string
+      translation: string | string[]
   ): TranslatableItem {
     const {
       dataFormat,
       contentType,
       interactionId,
       ruleType
     }: {
-      dataFormat?: string,
-      contentType?: string,
-      interactionId?: string,
-      ruleType?: string
-    } = this.stateAndContent[this.activeIndex] || {};
+      dataFormat: string,
+      contentType: string,
+      // The following two properties are set to null
```

Comment:
Same thing. Please don't just copy what @vojtechjelinek wrote, he was just giving you an example.